tf_fastmri_data/datasets/non_cartesian.py

```diff
@@ -81,7 +90,12 @@ def preprocessing(self, image, kspace):
                 traj[0],
             )
         traj = tf.repeat(traj, tf.shape(image)[0], axis=0)
-        orig_image_channels = ortho_ifft2d(kspace)
+        orig_image_channels = adjust_image_size(
```

Comment:
if we do the adjustment here, then let's remove it from the `nufft` function.

Reply:
But I am not sure exactly where would be the best place for it ...

Reply:
Hmmm, ill have a look.. I see minute issues in the way gpuNUFFT and kbNUFFT gives results at the moment.

Reply:
Fixed this now